libbeat/beat/beat.go

```diff
@@ -252,6 +254,11 @@ func (b *Beat) launch(bt Creator) error {
 		return err
 	}
 
+	err = b.loadTemplate()
```

Comment:
Hmm, if I understand it correctly, this means that the template loading is only attempted once at startup not on every connection made to ES? If this is the case, then this is a regression for two reasons:

* If ES is not up when the Beat start, the Beat will fail with an error. It used to "wait" for ES to come up
* If for whatever reason ES is restarted without data, but the Beat is not restarted, the template is not loaded again, resulting in bad mapping.